NORSE_actor/externalNORSERun.py

```diff
@@ -193,11 +207,17 @@
 	distribution = np.array(eng.extractDistribution(o)).tolist()
 	pBig = np.array(eng.extractPBig(o)).tolist()
 	xiBig = np.array(eng.extractXiBig(o)).tolist()
-
+	growthRate = density[i]*eng.extractGrowthRate(o)
+	runawayDensity = density[i]*eng.extractFraction(o)
+	runawayCurrent = runawayDensity * 1.6e-19 * 3e8 * np.sign(E_parallel[i])

```

Comment:
Constants should be defined in the object. Later these can be easily replaced by ITM constants.

Reply:
Done.